Makefile

```
@@ -127,3 +130,27 @@ lint_country_dialing_codes: update_pinpoint_supported_countries ## Checks that c
 
 check_asset_strings: ## Checks for strings
 	find ./app/javascript -name "*.js*" | xargs ./scripts/check-assets
+
+build_artifact: ## Builds zipped tar file artifact with IDP source code and Ruby/JS dependencies
```

We could also add ARTIFACT_DESTINATION_FILE as a target since that's the file made by this rule... then we could get fancy with some more Makefile magic

```suggestion
build_artifact: $(ARTIFACT_DESTINATION_FILE) ## Builds zipped tar file artifact with IDP source code and Ruby/JS dependencies

$(ARTIFACT_DESTINATION_FILE):
```